Massive Fixes

e-ee-ee opened the pull request:
- [NCNS] Japan and the Warlords should now have correct OOB's
- The Sichuan Famine will now end (the last event will remove the national spirit).
- The Marco Polo Bridge Incident Should happen at the right time.
- [NCNS] China is now able to annex the Northeast Army after the Xian incident.
- Japan's AI should now attempt to counter the Emperor's Conspiracy.
- The PRC will no longer create the People's Front on historical.
- [NCNS] RT56 Tree specific decisions will no longer show up on DLC tree RoC.
- The National Catastrophe has been tweaked to be more broadly applicable (no ideology locks) and now all Chinese Governments have access to it.
- Wang Jingwei's Branch on the RT56 tree has been updated to delay the Marco Polo Bridge Incident
- [NCNS] Fix state number conflict in goals: Control the Chinese capitals and in the goals regarding the mines in Northern Sweden

e-ee-ee committed:
This should fix Japan's OOB

e-ee-ee committed:
Updated WTT_China to fix Xian incident

e-ee-ee committed:
One last Xian related Fix

e-ee-ee committed:
Fixed marco bridge

e-ee-ee committed:
Fixed OOBs and a bug with famine

e-ee-ee committed:
clearing error log a bit

e-ee-ee committed:
Revert "Fixed OOBs and a bug with famine"

This reverts commit b4b45d01bdcb208b09245da9ae56d473529728b1.

e-ee-ee committed:
clreading various errors and fixing the undone commit

marijn211 commented on events/WTT_China.txt:
```diff
@@ -801,7 +801,7 @@ country_event = {
 		ai_chance = {
 			base = 30
 			modifier = {
-				CHI_is_on_historical_plan_trigger = yes
```
That's not equivalent and should probably be changed

e-ee-ee replied:
Pinged you about this in discord, but the trigger I removed was incorrectly copied over and does not work.

ilkober committed:
Less copper, ironed out

ilkober committed:
Fixing VSCode Germany error display

marijn211 commented:
@e-ee-ee Please mark bug threads as fixed after you fix them, if thats where you are getting the issues from

marijn211 committed:
Coal in eco laws

e-ee-ee committed:
Updated Japan's AI to better deal with the Emperor's Conspiracy

e-ee-ee committed:
Prevent the PRC from creating a faction on historical

e-ee-ee committed:
made RT56 decisions and missions not kill DLC tree games

e-ee-ee committed:
tweak to prevent blueshirts

e-ee-ee committed:
made the National Catastrope function as a shared branch for all Chinas

e-ee-ee committed:
fixed issues with the shared tree

e-ee-ee committed:
Merge branch 'Beta-Edits' into december-bugfixing-)

e-ee-ee committed:
fixed error log spammer

e-ee-ee committed:
fixed rest of error log spammer

e-ee-ee committed:
WJW can now delay kwantung army

e-ee-ee committed:
marco polo bridge will not happen if you start to surrender to China

BiccyThiccy committed:
POR - Support for Doctrine System + Tooltipping one event reward

marijn211 committed:
Port a bunch of missing GFX

marijn211 committed:
Fix unnecessary file override that caused bugs

marijn211 committed:
Fix several broken portraits without AAT

marijn211 committed:
Make NCNS show in the main screen DLC list

marijn211 committed:
Fix Huntziger having a loc collision

marijn211 committed:
Add a few missing naval dominance values

marijn211 committed:
Block Chen Du Xiu event for DLC PRC tree

marijn211 committed:
Add missing tactic

marijn211 reviewed (commented):
Error log contains some new low priority errors from the OOBs because the starting technologies of XSM and Japan do not match them

XSB needs arty tech
NSB Japan need tech and/or a design for light_tank_chassis_1

Thinking-waffle committed:
fix wrong state nanjing in japanese goal faction_goals_medium_term.txt

marijn211 commented:
@Thinking-waffle be aware SBG has fixed some other goals already on another PR

Thinking-waffle committed:
fixing state error for Vasterbotten faction_goals_medium_term.txt

Thinking-waffle committed:
fixing state error vasterbotten faction_goals_short_term.txt

Thinking-waffle committed:
Fixing Aden because we didn't add a state in south yemen faction_goals_short_term.txt

Thinking-waffle committed:
Fix Aden state number part two faction_goals_medium_term.txt

Thinking-waffle committed:
Fixing a bunch of state errors faction_goals_medium_term.txt

412 is changed into 2 different state in 2 different goals because we have more states in Iran. One checks for Hormuz, the other for oil.

There is an unwanted presence of the state number 479 in the AI guidelines of Baku, Iranian and Iraqi oilfields. Replacements need to be determined (ideally by Paradox)

katsuragiyy commented:
> @Thinking-waffle be aware SBG has fixed some other goals already on another PR

Only Suez related ones.

BiccyThiccy committed:
SEA - Importing some focus tree changes

BiccyThiccy committed:
PRC - Fixing the duplicate character errors

BiccyThiccy committed:
[NO-SEA JAP] - Reintroduce mistakenly removed event

marijn211 committed:
Merge branch 'Beta-Edits' into december-bugfixing-)

marijn211 committed:
Fix a few OOB errors

marijn211 committed:
Update XSM - Xibei San Ma.txt

marijn211 committed:
Clear a few errors

marijn211 committed:
Fix capitalization of a file part 1

marijn211 committed:
Fix capitalization of a file part 2

This reverts commit c4cf1f690318471a9e1f6efa6756cd8725572644.

marijn211 committed:
Make two Persian focuses more flexible to Rt56 possibilities
